tests/Library/Vanilla/Utility/DomUtilsTest.php

```diff
@@ -278,7 +278,7 @@ public function providePregReplaceCallbackTests(): array {
                 '<img src="img_test.jpg" alt="forbiddenword image" width="100" height="100">',
                 '<img src="img_test.jpg" alt="*** image" width="100" height="100">'],
             'emoji' => [1, '🤓','test 🤓', 'test ***'],
-            'count' => [3, 'a', '<p>a aaa is</p><p>aaa</p>', '<p>* *** is</p><p>***</p>']
+            'count' => [2, 'a+', '<p>a aaa is</p><p>aaa</p>', '<p>*** *** is</p><p>***</p>']
```

Comment:
My test was correct. There are three strings to be replaced so the function should return three.

Reply:
its currently returning number of replacement per tag, <p>a aaa is</p><p>aaa</p>' would count as 2.